JitsiConference.js

```diff
@@ -2067,12 +2062,8 @@ JitsiConference.prototype._acceptJvbIncomingCall = function(
         serverRegion);
 
     this._maybeClearSITimeout();
-    Statistics.sendAnalytics(createJingleEvent(
-        ACTION_JINGLE_SI_RECEIVED,
-        {
-            p2p: false,
-            value: now
-        }));
+
+    Statistics.sendAnalytics(createJingleEvent(ACTION_JINGLE_SI_RECEIVED, { p2p: false }));
```

Comment:
Why remove the timestamp from the event?

Reply:
Because it was never a timestamp :-) performance.now() does not return timestamps, it produces monotonically increasing marks from an arbitrary point in time, so only the difference between them is relevant. Also, some actions had it some didn't so I opted for that. Last, we are not actually using this event so its usefulness is questionable I guess.